src/handlers/image.rs

```diff
@@ -47,43 +47,51 @@ pub async fn image_handler(
     match determine_img_path(get_original_path().to_str().unwrap(), id) {
         Err(_) => (),
         Ok(path) => {
-            return image_handler_helper(id, path.to_str().unwrap(), query.0, CacheBehavior::Normal)
+            return image_handler_helper(
+                id,
+                path.to_str().unwrap(),
+                query.0,
+                CacheBehavior::Normal,
+            );
```

Comment:
Wieso ändert sich hier nichts bis auf das formatting? Sollte das formatting in Rust nicht durch rustfmt einheitlich sein und hier schon gepasst haben?

Reply:
Hab per rustfmt formatiert

Reply:
;

Reply:
Lag am ;